acme/acme/crypto_util.py

```diff
@@ -161,31 +160,23 @@ def _pyopenssl_cert_or_req_san(cert_or_req):
     :rtype: `list` of `unicode`
 
     """
-    # constants based on implementation of
-    # OpenSSL.crypto.X509Error._subjectAltNameString
-    parts_separator = ", "
+    # constants based on PyOpenSSL certificate/CSR text dump
```

Comment:
let's explain here why we use text dump - reference relevant issues, mention dep version requirements, maybe this PR - so that in future someone doesn't try to waste time to rewrite it